XamlControlsGallery/ControlPages/StandardUICommandPage.xaml.cs

```diff
@@ -96,7 +96,12 @@ private void ListViewRight_ContainerContentChanging(ListViewBase sender, Contain
         {
             MenuFlyout flyout = new MenuFlyout();
             ListItemData data = (ListItemData)args.Item;
-            MenuFlyoutItem item = new MenuFlyoutItem() { Command = data.Command};
+            MenuFlyoutItem item = new MenuFlyoutItem() { Command = data.Command };
+            flyout.Opened += delegate (object element, object e) {
+                var flyOutElement = element as MenuFlyout;
```

Comment:
```suggestion
                MenuFlyout flyoutElement = element as MenuFlyout;
```

You're casting your variable to a MenuFlyout anyway, so might as well declare it as such for clarity. And I would update the capitalization in the name to better align with the control's capitalization (Flyout).

Reply:
I agree, that makes the code more readable. Was probably a relic of refactoring.